app/models/category.rb

```diff
@@ -0,0 +1,10 @@
+class Category < ApplicationRecord
+  belongs_to :user
+  has_and_belongs_to_many :expenses
+
+  has_one_attached :image
+
```

Comment:
- [ ] This validation is not working. As name and icon both are mandatory for the category but I skipped the icon and still get the category get saved and appear on homepage. kindly consider fixing it.
requirement: The user fills out a form to create a new category, indicating their name and icon (both mandatory).

https://github.com/coderacheal/cent/assets/96683843/60b315c2-4d03-42ae-9df5-920e5dd8c93a